x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -158,7 +165,7 @@ func (k msgServer) RelayPayment(goCtx context.Context, msg *types.MsgRelayPaymen
 		// if they failed (one relay should affect all of them). From here on, every check will
 		// fail the TX ***
 
-		totalCUInEpochForUserProvider := paymentHandler.AddEpochPayment(ctx, relay.SpecId, epochStart, project.Index, providerAddr, relay.CuSum, strconv.FormatUint(relay.SessionId, 16))
+		totalCUInEpochForUserProvider := k.Keeper.AddEpochPayment(ctx, relay.SpecId, epochStart, project.Index, relay.Provider, relay.CuSum, relay.SessionId)
```

Comment:
we dont need the cache here anymore?

Reply:
done. see https://github.com/lavanet/lava/commit/82daa9388dc95b5c1dc4b22ba1c592ce0edef1c5